src/auto-evo/selection_pressure/AvoidPredationSelectionPressure.cs

```diff
@@ -25,6 +25,9 @@ public AvoidPredationSelectionPressure(Species predator, float weight) : base(we
             CommonMutationFunctions.Direction.Rear),
         new MoveOrganelleBack(organelle => organelle.HasSlimeJetComponent),
         new MoveOrganelleBack(organelle => organelle.HasMovementComponent),
+        new UpgradeOrganelle(organelle => organelle.HasSlimeJetComponent, "mucocyst"),
```

Comment:
Shouldn't this use the magic constant from the slime jet component? `MUCOCYST_UPGRADE_NAME` (you'll need to change it t public).